### ISSUE TITLE
Bug: Sonoma "text cursor" is treated as a window

### ISSUE THREAD

**Issue by nasyxx** [nikitabobko/AeroSpace#53]
I tried `watch -n 0.5 aerospace list-apps` to find the app-id, but failed.

The "text cursor":

<img width="322" alt="Screenshot 2023-12-14 at 1 56 16 AM" src="https://github.com/nikitabobko/AeroSpace/assets/5007613/114105f0-4673-4d9a-a5b1-9eff5cfe1da4">

https://discussions.apple.com/thread/255159017?sortBy=best

https://discussionschinese.apple.com/thread/255178136?sortBy=best

I don't want to turn off this feature because I need to use a multi-language and multi-input method environment in normal use, which is very useful.

**Comment by nikitabobko** [comment]
Is the "text cursor" window gets tiled on your machine? (on my machine it's considered as floating window)

**Comment by nasyxx** [comment]
> Is the "text cursor" window gets tiled on your machine?

Yes.

I believe I may have identified the reason. The issue might stem from the "text cursor" being treated as another current application window. I manually configured Aerospace to tile the application, which inadvertently may have included the "text cursor".

A straightforward way to replicate this issue is observed when using 

`run = ["layout tiling", "move-node-to-workspace term"]`

In this scenario, the "text cursor" is treated as one of the tiled windows. However, when using only 

`run = ["move-node-to-workspace term"]`

the app wezterm gets tiled while the "text cursor" remains floating.

The reason I added layout tiling is because I treat all windows as floating by default, and only certain applications are managed under this setting.


```toml
# others ...

#  all floating
[[on-window-detected]]
  if.app-name-regex-substring = ".*"
  check-further-callbacks = true
  run = "layout floating"

# others ...

## emacs window
[[on-window-detected]]
  if.app-id = "org.gnu.Emacs"
  run = ["layout tiling", "move-node-to-workspace emacs"]

## terminal window
[[on-window-detected]]
  if.app-id = "com.github.wez.wezterm"
  run = ["layout tiling", "move-node-to-workspace term"]

# others ...
```

**Comment by nikitabobko** [comment]
It's an AeroSpace bug. AeroSpace should detect "text cursor" and don't treat it as a window

**Comment by nikitabobko** [comment]
Released in 0.7.0-Beta
